src/schoology-extractor/data_extractor/lib/__init__.py

```diff
@@ -0,0 +1 @@
+__version__ = '0.1.0'
```

Comment:
Each Python file needs to have the file header on it. Please check that you have the recommended psionic header extension in Visual Studio Code (exact name is in the `.vscode/extensions.json` file). With that extension, every new file you create from within VS Code will automatically get the correct header. A file created outside of VS code will require manual editing. You can pull up the command palette and find the "insert header" command.

Does not apply to configuration files including *.json, ini-type config files, or xml config files.